### PR TITLE
Add additional fields import for Venues and Organizers

### PR DESCRIPTION
**Pull request by andrasguseo**
Add filter and logic to handle the saving of additional data for Organizers and Venues when doing a CSV import.

[TEC-4846]

[TEC-4846]: https://theeventscalendar.atlassian.net/browse/TEC-4846?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Tribe/Importer/File_Importer_Venues.php [hunk 1]
```diff
@@ -140,14 +140,39 @@ private function build_venue_array( $venue_id, array $record ) {
...
+		 * @var array $additional_venue_fields
+		 */
+		$additional_venue_fields = apply_filters( 'tribe_events_csv_import_venue_additional_fields', [] );
+
+		if ( ! empty ( $additional_venue_fields ) ) {
+			foreach ( $additional_venue_fields as $key => $csv_column ) {
+				$value = $this->get_value_by_key( $record, $key );
+				/**
+				 * This is needed when adding custom fields to the post type.
+				 * On save the metakey gets the "_Venue" prefix. It should be removed before the import.
```
Comment (Camwyn) [inline src/Tribe/Importer/File_Importer_Venues.php:157]:
```suggestion
				 * On save the metakey gets the "_Venue" prefix added. So it should be removed before the import.
```

**Review by Camwyn — CHANGES_REQUESTED** [review]
Some docblock tweaks, otherwise 👨🏼‍🍳

File: src/Tribe/Importer/File_Importer_Organizers.php [hunk 2]
```diff
@@ -96,6 +96,31 @@ private function build_organizer_array( $organizer_id, array $record ) {
 		// Handle the manual stuff.
 		$organizer['FeaturedImage'] = $this->get_featured_image( $organizer, $record );
 
+		/**
+		 * Filter to allow the saving of additional fields for Organizers.
+		 * - key: the metakey (postmeta table) or column name (posts table)
+		 * - value: the CSV column ID from the column mapping
+		 *
+		 * @var array $additional_organizer_fields
+		 */
```
[the remark this answers is not in the document]
Comment (andrasguseo) [reply src/Tribe/Importer/File_Importer_Organizers.php:105]: Good learning, thanks!

**Comment by andrasguseo** [comment]
@Camwyn Thanks for the code review and the feedback. So much appreciated!
There was a conflict in the readme because a release went out since the branch was created. I "tried" to resolve it, please check if I did it correctly. Thanks again!

**Comment by Camwyn** [comment]
This is scheduled to go out in Sprint 32 (we're currently on 29) I'll keep it up to date until then. Self-assigning

**Comment by andrasguseo** [comment]
Awesome, thanks @Camwyn !

**Comment by andrasguseo** [comment]
This was handled in two other PRs, so closing it.
* https://github.com/the-events-calendar/the-events-calendar/pull/4702
* https://github.com/the-events-calendar/the-events-calendar/pull/4738

**Pull request closed without merging**